### DISCUSSION TITLE
Support retrying by updating the workflow CR spec?

### DISCUSSION THREAD

**Discussion by maxsxu** [argoproj/argo-workflows#12027] [Ideas]
Is it possible to support retrying a workflow by updating the workflow CR spec?

**Comment by terrytangyuan** [comment]
Technically yes since the current implementation of retrying is by modifying the workflow status, spec, etc.

**Reply by agilgur5** [reply]
Correction here, it actually currently isn't, although it _should_ be in the way Terry described.

See #12538 which would make this possible and fix the current broken separation.

**Reply by maxsxu** [reply]
Thanks for clarify!
